claim/apps.py

```diff
@@ -58,6 +58,7 @@ class ClaimConfig(AppConfig):
     claim_validation_multiple_services_explanation_required = True
     # Provide absolute path to autogenerating function for claim code
     autogenerate_func = 'claim.utils.autogenerate_nepali_claim_code'
```

Comment:
Can you move these config fields into the config json, change the default values to `None`, and add proper initialization? (you can copy this initializer and remove initialization by hand https://github.com/openimis/openimis-be-payment_cycle_py/blob/bb63f5060049c662818e34be3cef4450d481c24f/payment_cycle/apps.py#L31)

The reason is currently these fields are not updatable on server by providing the config json in admin panel. These fields can only be updated by updating the code and redeploying the backend.